### PR TITLE
refactor: Added components are unused saying message in builder panel when builder layout is empty

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Added message when none of the component not used in builder layout

### Screenshots
- https://a.cl.ly/9Zu8yx4Q

### Types of changes
- Non-breaking changes
- Removed extra unwanted code
- Then conditionally loaded span for this message

### How has this been tested?
- With blank builder ;layout as per above screenshot

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/assets/css/ast-builder-customizer.css [hunk 1]
```diff
@@ -636,6 +636,16 @@
     box-shadow: none !important;
 }
 
+.ahfb-available-items-pool- .ahfb-all-coponents-unused  {
```
Comment (vrundakansara) [inline inc/assets/css/ast-builder-customizer.css:639]: I think we must rename this class `ahfb-available-items-pool-`.

File: inc/customizer/extend-custom-controls/src/draggable/draggable-component.js [hunk 2]
```diff
@@ -119,11 +74,22 @@ const DraggableComponent = props => {
...
+		});
+	});
+
 	return <div className="ahfb-control-field ahfb-available-items">
 		<div className="ahfb-available-items-pool-">
 			{Object.keys(choices).map(item => {
 				return renderItem(item, 'links');
 			})}
+			{!droppedCount &&
+				<span className="ahfb-all-coponents-unused"> {__('Components Being Used in Builder will Visible Here.', 'astra')} </span>
```
Comment (vrundakansara) [inline inc/customizer/extend-custom-controls/src/draggable/draggable-component.js:91]: `Elements used in the builder will be visible here.`
Let's reframe the message too.

Comment (vrundakansara) [inline inc/customizer/extend-custom-controls/src/draggable/draggable-component.js:91]: Do you think we must remove the `ahfb` prefix altogether?

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: inc/customizer/extend-custom-controls/src/draggable/draggable-component.js (hunk 2 above)
Comment (imnavanath) [reply inc/customizer/extend-custom-controls/src/draggable/draggable-component.js:91]: Replaced with - `ast-builder-elements-notice`

Comment (imnavanath) [reply inc/customizer/extend-custom-controls/src/draggable/draggable-component.js:91]: Done 👍

File: inc/assets/css/ast-builder-customizer.css (hunk 1 above)
Comment (imnavanath) [reply inc/assets/css/ast-builder-customizer.css:639]: Updated with - `ast-builder-elements-section`

File: inc/customizer/extend-custom-controls/src/draggable/draggable-component.js (hunk 2 above)
Comment (vrundakansara) [reply inc/customizer/extend-custom-controls/src/draggable/draggable-component.js]: @imnavanath - Let's not capitalize all the words. Keep it `Elements used in the builder will be visible here.`.

Comment (imnavanath) [reply inc/customizer/extend-custom-controls/src/draggable/draggable-component.js]: Corrected 👍

**Review by vrundakansara — APPROVED** [review]

File: inc/customizer/extend-custom-controls/src/draggable/draggable-component.js [hunk 3]
```diff
@@ -88,25 +60,8 @@ const DraggableComponent = props => {
...
-			}} className={'ahfb-builder-item-start ahfb-move-item'} list={theitem}
-						   setList={newState => onDragEnd(newState)}>
-				<div className="ahfb-builder-item" data-id={item}
-					 data-section={choices[item] && choices[item].section ? choices[item].section : ''} key={item}>
-								<span className="ahfb-builder-item-icon ahfb-move-icon">
-									dangerouslySetInnerHTML={{ __html: Icons['drag'] }}
-								</span>
-					{choices[item] && choices[item].name ? choices[item].name : ''}
-				</div>
-			</ReactSortable>}
```
Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/draggable/draggable-component.js:109]: @imnavanath Can you please explain why are we removing this? I think using `droppedCount` only we can check the number of dropped elements and add the message?

Comment (imnavanath) [reply inc/customizer/extend-custom-controls/src/draggable/draggable-component.js:109]: This code is not required, this one is extra piece of code.
You can see here - https://github.com/brainstormforce/astra/pull/2436/files/a7200d3a2be92a4e73e602865a26ad90350a0fcc#diff-d7139873fc188a4eea4038ebfe85db88b5d95f8b8fa4857cf49d5ec5c41fc533R88 we just called 'links' builder components,  for 'available' we don't need. Previously it was for to showcase remaining components in builder panel. But now, it's not required.

**Review by premanshup — APPROVED** [review]

**Pull request merged**